### PR TITLE
ARROW-5325: [Archery][Benchmark] Output properly formatted jsonlines from benchmark diff cli command

### PR DESCRIPTION
**Pull request by kszucs**

### CODE REVIEW

**Comment by kszucs** [comment]
Wanted to merge it to my own fork, accidentally created for apache/arrow. Closing.

**Comment by kszucs** [comment]
@ursabot ping

**Comment by kszucs** [comment]
@ursabot build

**Comment by fsaintjacques** [comment]
@ursabot benchmark

**Comment by wesm** [comment]
Should the bot post a link to the builds?

**Comment by kszucs** [comment]
Not yet, I'm working on the [comment reporter](https://github.com/ursa-labs/ursabot/pull/66).

**Comment by kszucs** [comment]
This message is produced by the hook immediately after it receives the event, so it's just an acknowledgement. The "result" will be sent by the reporter.

**Comment by kszucs** [comment]
@ursabot build

**Comment by kszucs** [comment]
@ursabot benchmark

**Comment by kszucs** [comment]
@ursabot benchmark

**Comment by kszucs** [comment]
@ursabot benchmark

**Comment by kszucs** [comment]
@ursabot benchmark

**Comment by kszucs** [comment]
@ursa-labs/team finally it works

**Comment by kszucs** [comment]
@ursabot benchmark

**Comment by kszucs** [comment]
@ursabot benchmark

File: cpp/src/arrow/compute/kernels/aggregate-benchmark.cc [hunk 1]
```diff
@@ -18,22 +18,16 @@
 #include "benchmark/benchmark.h"
 
 #include <vector>
-#ifdef _MSC_VER
-#include <intrin.h>
-#else
-#include <immintrin.h>
-#endif
```
Comment (kszucs) [inline cpp/src/arrow/compute/kernels/aggregate-benchmark.cc]: Arm benchmark fails otherwise.

**Comment by wesm** [comment]
This seems to only need a clang-format to pass

**Comment by kszucs** [comment]
@ursabot benchmark

**Comment by kszucs** [comment]
The ARM benchmark fails, I've rebuilt the image (pandas/numpy install took more than an hour), now testing it.

**Comment by kszucs** [comment]
@ursabot benchmark

**Comment by kszucs** [comment]
@ursabot benchmark

**Comment by kszucs** [comment]
@ursabot benchmark

**Review by kszucs — COMMENTED** [review]
+1

**Pull request closed without merging**